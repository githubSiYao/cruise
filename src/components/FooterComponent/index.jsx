import React from 'react'
import './index.scss'

export default class FooterComponent extends React.PureComponent {
  render() {
    return (
      <footer className="footer">
        Â© Copyright 2019 ThoughtWorks
      </footer>
    )
  }
}