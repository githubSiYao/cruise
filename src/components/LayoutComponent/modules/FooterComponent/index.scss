.footer {
  @media screen and (max-width: 1024px) {
    display: none;
  }
  position: fixed;
  bottom: 0;
  width: 100%;
  text-align: center;
  padding: 4px;
  z-index: 1;
  background-color: #ffffff;
  box-shadow: 1px -4px 10px rgba(0, 0, 0, 0.1);
}
