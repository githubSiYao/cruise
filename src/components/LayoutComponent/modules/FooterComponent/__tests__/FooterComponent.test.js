import React from 'react';
import renderer from 'react-test-renderer';
import FooterComponent from '../index';

describe('<FooterComponent />', () => {
  it('FooterComponent Snapshot', () => {
    const component = renderer.create(<FooterComponent/>);
 
    let snapshot = component.toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});