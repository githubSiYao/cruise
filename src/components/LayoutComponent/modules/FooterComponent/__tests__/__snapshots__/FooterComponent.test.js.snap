// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterComponent /> FooterComponent Snapshot 1`] = `
<footer
  className="footer"
>
  Â© Copyright 2019 ThoughtWorks
</footer>
`;
